autoPyTorch/pipeline/components/training/trainer/base_trainer.py

```diff
@@ -537,8 +537,8 @@ def get_hyperparameter_search_space(
         dataset_properties: Optional[Dict] = None,
         weighted_loss: HyperparameterSearchSpace = HyperparameterSearchSpace(
             hyperparameter="weighted_loss",
-            value_range=[True, False],
-            default_value=True),
+            value_range=[1],
```

Comment:
```suggestion
            value_range=(1, ),
```

Reply:
to check, if passing a tuple is what is expected.